blockprotocol/blockprotocol
# New block dev package: mock-block-dock

This PR introduces a package for making it easier to develop blocks, `mock-block-dock` - which is essentially a dummy embedding application.

It provides an in-memory datastore, and passes the basic Block Protocol functions to blocks, e.g. `createEntities`, `createLinks`, etc, so that block authors don't have to provide their own mock embedding application.

Usage:
```jsx
<MockBlockDock>
  <TestBlock {...props} />
</MockBlockDock>
```

As part of the PR, I've removed the existing mock functions when rendering a block on the site, in favour of this package.

To test:
1. Run the site locally (`yarn dev`)
2. Go to `/hub` and click on a block (e.g. `Image`)
3. Add an image, check it works
or
1. Go to some block in the `hash` repo, e.g. image, video, embed
2. `yarn add mock-block-dock`
3. in the dev renderer, remove the functions passed to the component and wrap it in this package instead
